irods_browser_app/static/css/site_base_irods.css

```diff
@@ -8,10 +8,6 @@
     opacity: 0.7;
 }
 
-#irodsContent .modal-lg {
-    width: 570px;
-}
-
```

Comment:
This class was not needed after new style definitions. I have removed the references now.